openapi/components/schemas/AuthorizationsProperties.yaml

```diff
@@ -0,0 +1,19 @@
+authorizations:
+  type: array
```

Comment:
great to see this as an array since a given DRS object may have multiple auth approaches... e.g. Gen3 bearer token for dev access, RAS Passport for researcher access